lightning/cogs/automod/ui.py

```diff
@@ -191,3 +203,338 @@ async def warn_ban(self, itx: discord.Interaction, button: discord.ui.Button):
         self.choice = "warn_ban"
         await itx.response.edit_message(view=None)
         self.stop()
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider adding user feedback before stopping the view.

It's good practice to send a confirmation message or update the current message to inform the user that their choice has been registered before stopping the view.

```suggestion
    async def warn_kick(self, itx: discord.Interaction, button: discord.ui.Button):
        self.choice = "warn_ban"
        await itx.response.edit_message(content="Your choice has been registered. Processing...", view=None)
        self.stop()
```